app/controllers/donations_controller.rb

```diff
@@ -33,13 +33,16 @@ def index
     # Using the @donations allows drilling down instead of always starting with the total dataset
     @donations_quantity = @donations.collect(&:total_quantity).sum
     @paginated_donations_quantity = @paginated_donations.collect(&:total_quantity).sum
+    @items = current_organization.items.alphabetized.select(:id, :name)
     @total_value_all_donations = total_value(@donations)
     @paginated_in_kind_value = total_value(@paginated_donations)
     @total_money_raised = total_money_raised(@donations)
     @storage_locations = @donations.filter_map { |donation| donation.storage_location if !donation.storage_location.discarded_at }.compact.uniq.sort
     @selected_storage_location = filter_params[:at_storage_location]
     @sources = @donations.collect(&:source).uniq.sort
     @selected_source = filter_params[:by_source]
+    @selected_item = filter_params[:by_item_id]
+    @donation_totals = DonationTotalsService.call(current_organization.donations.class_filter(scope_filters))
```

Comment:
Wait, what does this even have to do with this PR? It's not even used anywhere that I can tell.